src/azure-cli/azure/cli/command_modules/network/commands.py

```diff
@@ -897,7 +902,8 @@ def _make_singular(value):
         g.generic_update_command('update', child_collection_prop_name='frontend_ip_configurations',
                                  getter_name='lb_get',
                                  getter_type=network_load_balancers_custom,
-                                 setter_name='begin_create_or_update',
+                                 setter_name='update_lb_frontend_ip_configuration_setter',
```

Comment:
Better to use verb as setter_name such as `update_lb_frontend_ip_configuration` instead of `update_lb_frontend_ip_configuration_setter` as it's a nouns.

Reply:
I'm following the existing setters they all have `_setter` suffix.